src/app/map/suitability-map-panel/suitability-map-panel.component.html

```diff
@@ -4,16 +4,15 @@
 
     <section class="map__control__section">
         <h2 class="map__control__title title">
-            <span class="title__text">{{ 'SUITABILITY.TITLE' | translate}}</span>
+            <span class="title__text">{{ 'SUITABILITY.TITLE' | translate }}</span>
             <span class="pull-right title__icon title__icon--actionable" (click)="onHideButtonClick($event)">
                 <i class="glyphicon glyphicon-chevron-left" placement="bottom" tooltip="Hide"></i>
             </span>
         </h2>
 
         <div class="map__control__body">
             <p>
-                <!-- Shows what crops are best to plant in a specific area to attain maximum yield. -->
-                {{ 'SUITABILITY.DESCRIPTION' | translate}}
+               {{ 'SUITABILITY.DESCRIPTION' | translate}}
```

Comment:
One more space. lol. Add space after the translate pipe like this `{{ 'SUITABILITY.DESCRIPTION' | translate }}`